.goreleaser.yml

```diff
@@ -104,7 +104,7 @@ archives:
   format: binary
   builds:
   - prom-migrator
-  name_template: "{{ .Binary }}_0.0.3_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}"
```

Comment:
Could you create an issue to move this hardcoded value out of `.goreleaser.yml`? We shouldn't change this file on each prom-migrator release.